core/types/calls.cc

```diff
@@ -746,7 +876,7 @@ DispatchResult dispatchCallSymbol(const GlobalState &gs, DispatchArgs args,
                         consumed.insert(arg);
 
                         TypeAndOrigins tpe;
-                        tpe.origins = args.args.back()->origins;
+                        tpe.origins = {kwargsLoc};
```

Comment:
It looks like we're dropping the original TypeAndOrigins information on the floor here, is that right?

There was a TypeAndOrigin for the last arg, and we're just taking it's loc. Way up above, where we first initialize kwargsLoc, can we also initialize something like kwargsTypeAndOrigin using the `args.args` list of TypeAndOrigin's?

Reply:
Sure, I can make a TypeAndOrigins up above instead, and use that for tracking the location and type of the kwargs hash.

Reply:
I'm going to punt on this for now, and leave a TODO about trying to find the origins from the send args when the keyword args were inlined.